### PR TITLE
brev 0.6.17 (new formula)

### PR DESCRIPTION
**Pull request by r33drichards**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/brev.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  desc "CLI tool for managing workspaces provided by brev.dev"
+  homepage "https://docs.brev.dev"
+  url "https://github.com/brevdev/brev-cli/archive/refs/tags/v0.6.17.tar.gz"
+  sha256 "cf69be0558cd87ca5838372aeca6b85fe937f1bd8eb44c03a88355ca088aa9ea"
+  license "MIT"
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v#{version}"
+    system "go", "build", *std_go_args(output: bin/"brev", ldflags: ldflags)
```
Comment (carlocab) [inline Formula/brev.rb:11]:
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags)
```

File: Formula/brev.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Brev < Formula
+  desc "CLI tool for managing workspaces provided by brev.dev"
+  homepage "https://docs.brev.dev"
+  url "https://github.com/brevdev/brev-cli/archive/refs/tags/v0.6.17.tar.gz"
+  sha256 "cf69be0558cd87ca5838372aeca6b85fe937f1bd8eb44c03a88355ca088aa9ea"
+  license "MIT"
+  depends_on "go" => :build
```
Comment (carlocab) [inline Formula/brev.rb:7]:
```suggestion
  license "MIT"

  depends_on "go" => :build
```

File: Formula/brev.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  def install
+    ldflags = "-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v#{version}"
+    system "go", "build", *std_go_args(output: bin/"brev", ldflags: ldflags)
+  end
+
+  test do
+    system "#{bin}/brev", "healthcheck"
```
Comment (carlocab) [inline Formula/brev.rb:15]: Does this return a non-zero exit code if there's an issue)

Comment (r33drichards) [reply Formula/brev.rb:15]: yes
```
λ brev healthcheck
Healthy!
λ echo $?
0
λ brev healthcheck
Not Healthy!
Error: [error] github.com/brevdev/brev-cli/pkg/cmd/healthcheck/healthcheck.go:34 
	: [error] github.com/brevdev/brev-cli/pkg/cmd/healthcheck/healthcheck.go:47 
	: [error] github.com/brevdev/brev-cli/pkg/store/healthcheck.go:10 
	: Get "https://ade5dtvtaa.execute-api.us-east-1.amazonaws.com/api/health?utm_source=cli": dial tcp: lookup ade5dtvtaa.execute-api.us-east-1.amazonaws.com: Temporary failure in name resolution
Usage:
  brev healthcheck [flags]

Examples:
brev healthcheck

Flags:
  -h, --help   help for healthcheck

Global Flags:
      --version   Print version output

λ echo $?
1
```

File: Formula/brev.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags)
+  end
+
+  test do
+    system "#{bin}/brev", "healthcheck"
```
Comment (carlocab) [inline Formula/brev.rb:16]: Great, but we don't need interpolation here.
```suggestion
    system bin/"brev", "healthcheck"
```

**Comment by r33drichards** [comment]
should I squash my commits?

**Comment by carlocab** [comment]
Yes please; thank you!

**Comment by r33drichards** [comment]
You're welcome! all squashed :hammer:

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**